tox.ini

```
@@ -19,14 +19,15 @@ ignore=
 
 per-file-ignores=
     ; TYPE_CHECKING block suggestions
-    tests/*: TC001, TC002, TC003
+    tests/*: A, B, B904, C, E731, TC, ISC, N, SIM
+
     ; for clarity we don't merge 'with Conf():' context trees
-    tests/unit/parsec/*: SIM117, TC001, TC002, TC003
+    tests/unit/parsec/*: B, B904, C, SIM, TC
```

I think the B904 is redundant?

```suggestion
    tests/unit/parsec/*: B, C, SIM, TC
```